src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceDao.java

```diff
@@ -196,22 +196,27 @@ public void deleteProject(UUID id) {
     List<UUID> projectIds =
         projectRefs.stream().map(ProjectRefs::getProjectId).collect(Collectors.toList());
     if (projectIds.size() > 0) {
-      MapSqlParameterSource markParams =
-          new MapSqlParameterSource().addValue("project_ids", projectIds);
-
-      final String sqlMarkProjects =
-          "UPDATE project_resource SET marked_for_delete = true" + " WHERE id IN (:project_ids)";
-      jdbcTemplate.update(sqlMarkProjects, markParams);
-
-      final String sqlMarkBuckets =
-          "UPDATE bucket_resource SET marked_for_delete = true"
-              + " WHERE project_resource_id IN (:project_ids)";
-      jdbcTemplate.update(sqlMarkBuckets, markParams);
+      markProjectsForDelete(projectIds);
     }
 
     return projectIds;
   }
 
+  @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
+  public void markProjectsForDelete(List<UUID> projectIds) {
+    MapSqlParameterSource markParams =
+        new MapSqlParameterSource().addValue("project_ids", projectIds);
+
+    final String sqlMarkProjects =
+        "UPDATE project_resource SET marked_for_delete = true" + " WHERE id IN (:project_ids)";
```

Comment:
I know you're just moving code around but I think the string concatenations can be removed...these are remnants of switching to google code style